addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -348,6 +350,8 @@ func _on_input_event(_viewport: Node, event: InputEvent, _shape_idx: int):
 		MOUSE_BUTTON_LEFT:
 			if I.active:
 				await on_item_used(I.active)
+				# after item has been used return to normal state
+				I.active = null
```

Comment:
Is this needed? You're already changing the current active inventory item inside the `on_item_used` function.

Reply:
No it is not needed I just checked it and on_item_used() is called just before the I.active = null so it is not required. I removed the line and tested and it works as expected.